pom.xml

```diff
@@ -123,6 +123,26 @@
             <artifactId>pac4j-jwt</artifactId>
             <version>${pac4j.version}</version>
         </dependency>
+        <dependency>
```

Comment:
Adding all libraries not to use them but to check there is no incompatibility...